cookie-banner-rules-list.json

```diff
@@ -3791,8 +3715,9 @@
     },
     {
       "click": {
-        "optIn": "span#hf_cookie_text_cookieAccept",
-        "presence": "div.pre-cookie-modal-body"
+        "optIn": ".nds-btn.dialog-actions-accept-btn.css-60b779.ex41m6f0.btn-primary-dark.btn-md",
+        "optOut": ".nds-btn.dialog-actions-decline-btn.css-60b779.ex41m6f0.btn-primary-dark.btn-md",
+        "presence": ".dialog-content-wrapper.dialog-appear-done.dialog-enter-done"
```

Comment:
Please don't use styling classes for the selector as it may break when the style changes. Would'nt e.g. `.dialog-actions-accept-btn` be enough?